tests/integration/targets/rds_instance/roles/rds_instance/tasks/test_complex.yml

```diff
@@ -139,9 +139,6 @@
         port: 1150
         max_allocated_storage: 150
       register: result
-      retries: 30
-      delay: 10
-      until: result is not failed
```

Comment:
It probably wouldn't be noticeable consistently - it's intended to be a safeguard for test flakiness if the waiter time isn't long enough. But if it turns into an issue for CI it could always be added back.